src/hotspot/share/opto/vectornode.cpp

```diff
@@ -1042,7 +1042,7 @@ Node* VectorNode::try_to_gen_masked_vector(PhaseGVN* gvn, Node* node, const Type
   }
 }
 
-bool VectorNode::should_swap_inputs_to_help_global_value_numbering() {
+bool VectorNode::is_commutative() {
```

Comment:
Why did you make this change? It seems unrelated, maybe it slipped in from another change set? Plus, it is not an accurate name, if `if (in(1)->_idx > in(2)->_idx) {` would fail, you would say that it is not commutative ... which is wrong ;)